include/picongpu/plugins/openPMD/openPMDWriter.hpp

```diff
@@ -373,23 +377,40 @@ make sure that environment variable OPENPMD_BP_BACKEND is not set to ADIOS1.
                 }
             case ConfigurationVia::CommandLine:
```

Comment:
Note that we don't need to fix the `ConfigurationVia::Toml` path since checkpoints cannot be configured via TOML.